internal/action/generate.go

```diff
@@ -161,7 +161,7 @@ func hasPwRuleForSecret(name string) (string, pwrules.Rule) {
 
 // generatePassword will run through the password generation steps.
 func (s *Action) generatePassword(ctx context.Context, c *cli.Context, length, name string) (string, error) {
-	if domain, rule := hasPwRuleForSecret(name); domain != "" {
+	if domain, rule := hasPwRuleForSecret(name); domain != "" && !c.Bool("force") {
```

Comment:
Well, I'm not sure that's really the behavior we want for `force` in generate, is it?

As I understand it, `force` is supposed to overwrite the existing entry even if it already exists, no?
So, if for example I need to generate a new password for a website because it was recently compromised, I might want to use the force flag, but I would still like it to respect the password rules for that website, no?

Reply:
Yes. We could have another flag for that. More flags, more documentation, more maintenance but also more control.
I don't have a strong preference.

Reply:
I just double checked and the force flag isn't really needed to overwrite an existing password: we get a prompt instead:
https://github.com/gopasspw/gopass/blob/55ccbbb197e4c9dae2f8f8de6b8a8586353e014c/internal/action/generate.go#L60